src/Microsoft.XmlSerializer.Generator/dir.props

```diff
@@ -2,10 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <PreReleaseLabel>preview1</PreReleaseLabel>
-    <PackageVersion>1.1.0</PackageVersion>
+    <PreReleaseLabel>preview2</PreReleaseLabel>
```

Comment:
I think we can remove our own override of the PreReleaseLabel now so we will get it automatically from the branch-wide configured value. Can you please give it a try?